src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -49,6 +50,10 @@ const MSG = defineMessages({
     id: 'dashboard.ActionsPage.StakingWidget.objectButton',
     defaultMessage: 'Object',
   },
+  backButton: {
```

Comment:
we have a string already that we use for back buttons:
`{{ id: 'button.back' }}`